lib/shigeki.rb

```diff
@@ -1,24 +1,30 @@
 class Shigeki
-  attr_accessor :controller_name
+  attr_accessor :controller_name, :view_context
 
-  def initialize(controller_name)
+  def initialize(controller_name, view_context)
```

Comment:
This might be better as view_context: nil or view_context = nil, since not all use cases will need view_context

Then, down in `method_missing`'s `view_context.respond_to?(method)` check, make that into `view_context&.respond_to?(method)`